components/channel_header/__snapshots__/channel_header.test.jsx.snap

```diff
@@ -150,13 +150,23 @@ exports[`components/ChannelHeader should render active flagged posts 1`] = `
     />
     <HeaderIconWrapper
       ariaLabel={true}
-      buttonClass="style--none channel-header__icon"
+      buttonClass="style--none channel-header__icon wide"
       buttonId="channelHeaderPinButton"
       iconComponent={
-        <PinIcon
-          aria-hidden="true"
-          className="icon icon__pin"
-        />
+        <div
+          className="flex-child"
+        >
+          <span
+            className="icon__text"
+            id="channelMemberCountText"
+          >
+            -
```

Comment:
is this ok?
I'm guessing this is for having a `-` until it loads, but shoulnd't it be like:
```js
{'-'}
```

Reply:
@Willyfrog i guess it is getting displayed as `-` since we are using the value of the variable **countText**.

Reply:
if it doesn't get to be displayed, why adding it then?